Rakefile

```diff
@@ -16,6 +16,7 @@ namespace :build do
     ["x86_64-linux", "x86_64-redhat-linux"],
     ["x86_64-darwin", "x86_64-apple-darwin"],
     ["arm64-darwin", "aarch64-apple-darwin"],
+    ["arm64-linux", "aarch64-linux-gnu"],
```

Comment:
I think this should be `["aarch64-linux"`. I haven't seen `arm64-linux` so far and also your [Graviton output](https://github.com/sparklemotion/nokogiri/pull/2332#issuecomment-929548470) calls it `gem_platform: aarch64-linux`.

Reply:
Ah, of course. Will fix.

Reply:
Updated.